WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -60,7 +60,7 @@ class PointOfSaleAggregateModel: ObservableObject, PointOfSaleAggregateModelProt
 
     @Published private(set) var orderState: PointOfSaleOrderState = .idle
 
-    private var order: Order? = nil
+    private(set) var order: Order? = nil
```

Comment:
This should remain private – views should never need the order. `OrderState` should be enough, and if it's not the associated value can be updated to share more information.

I don't think you're actually reading this anywhere outside the aggregate model though...

Reply:
We don't, was some leftover of my previous attempt. Gone now!